core/pythonAction/pythonaction.py

```diff
@@ -61,7 +61,7 @@ def run():
     result = None
     try:
         exec(flask.g, namespace)
```

Comment:
I think this evolved. At some point there was an eval part of init. I'll fix it part of this PR.